cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -22,6 +22,9 @@ spec:
     # replicasPerFailureDomain: 2
     # The name of the failure domain to place further down replicas
     # subFailureDomain: host
+    # hybridStorage:
```

Comment:
How about moving this up to line 21? The hybrid settings seem more common than other settings to change.

Reply:
moved it above, to line 21